mesonbuild/backend/vs2010backend.py

```diff
@@ -983,9 +981,8 @@ def gen_vcxproj(self, target, ofname, guid):
                     else:
                         define = arg[2:]
                     # De-dup
-                    if define in file_defines[l]:
-                        file_defines[l].remove(define)
-                    file_defines[l].append(define)
+                    if define not in file_defines[l]:
+                        file_defines[l].append(define)
```

Comment:
This change also seems unrelated, but it's cleanup like the above change, so can be clubbed with it.

Reply:
Split cleanup into own commit.